spec/models/incoming_mail_handler_spec.rb

```diff
@@ -1,9 +1,8 @@
 describe "Handles incoming email" do
```

Comment:
I removed the commits for this for now -- will update separately since it was not strictly required for merge and this puppy is already quite large :dog: